tools/wpt/run.py

```diff
@@ -395,17 +395,18 @@ def setup_kwargs(self, kwargs):
                               device_serial=device_serial,
                               prompt=kwargs["prompt"])
 
-        if "ADB_PATH" not in os.environ:
-            adb_path = os.path.join(android.get_paths(None)["sdk"],
-                                    "platform-tools",
-                                    "adb")
-            os.environ["ADB_PATH"] = adb_path
-        adb_path = os.environ["ADB_PATH"]
+        if kwargs["adb_binary"] is None:
```

Comment:
Is `adb_binary` always set? If not we probably want to use `kwargs.get()` here.

Reply:
Yes, it comes from the command line args and we give everything a default value.